.circleci/config.yml

```diff
@@ -181,4 +154,6 @@ workflows:
               only:
                 - master
     jobs:
-      - linux_swift_4.2
+      - linux_on_docker:
+          name: linux_swift_4.2
+          image: norionomura/swift:swift-4.2-branch
```

Comment:
I'll open another PR updating daily to `swift-5.0-branch`, since it needs workarounds for crashing test on `swift-5.0-branch`.